src/content/fr/2019/cms.md

```diff
@@ -0,0 +1,662 @@
+---
+part_number: III
+chapter_number: 14
+title: CMS
+description: Chapitre CMS de l'Almanach Web 2019 couvrant l'adoption des CMS, la façon dont les solutions CMS sont construites, l'expérience utilisateur des sites web propulsés par les CMS et l'innovation des CMS.
+authors: [ernee, amedina]
+reviewers: []
+translators: [JustinyAhin]
+discuss: 
+published: 
+last_updated: 
```

Comment:
Leave as English date for now - will do a bulk update when French version goes live, but current timestamp will remind us of the format

```suggestion
last_updated: 2019-11-23T00:00:00.000Z
```